src/utils/staticPathProxy.ts

```diff
@@ -0,0 +1,22 @@
+export const staticPathProxy =
+  (proxyPrefix: string, fetch: typeof self.fetch = self.fetch) =>
+  async (url: string, request: Request) => {
+    const method = request.method;
+    const headers = new Headers(request.headers);
```

Comment:
should we just pass request.headers here?
Was this to update Host header, which probably we can't change in fetch()

Reply:
I think so, yeah, I'll push this in a sec